geometric_features/aggregation/seaice/historical_sea_ice.py

```diff
@@ -0,0 +1,33 @@
+def qgreenland(gf):
```

Comment:
@xylar , is there a coding style guideline for this repo about correspondence between function/class names and file names?